pkg/config/generate.go

```diff
@@ -409,7 +410,7 @@ func Generate(cs *acsapi.ContainerService, c *Config) (err error) {
 			// sync kubeconfig has the same capabilities as admin kubeconfig, only difference
 			// is the use of HCP internal DNS to avoid waiting for the Azure loadbalancer to
```

Comment:
Follow-up issue to remove this in favor of using a service account.

Reply:
not sure I understand:
a) sync effectively needs system:admin, so it might as well just use it
b) there's a bootstrapping issue whereby the first time sync runs, all there is is system:admin.

Reply:
Yeah, you are right about the boostrapping problem.